### PR TITLE
Adding darknet_ros to documentation index for distro

### PR DESCRIPTION
**Pull request by mbjelonic**
I'd like darknet_ros to be indexed and documented on ros.org.

### CODE REVIEW

**Comment by mbjelonic** [comment]
Hi, 
how should I continue with the failed checks? Thank you in advance!
Best,
Marko

File: kinetic/distribution.yaml [hunk 1]
```diff
@@ -1546,6 +1546,25 @@ repositories:
...
+      type: git
+      url: https://github.com/leggedrobotics/darknet_ros
+      version: master
+    release:
+      packages:
+      - darknet_ros
+      - darknet_ros_msgs
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/leggedrobotics/darknet_ros-release.git
```
Comment (dirk-thomas) [inline kinetic/distribution.yaml]: This repository doesn't seem to exist?

Comment (mbjelonic) [reply kinetic/distribution.yaml]: Sorry for the mistake. The repository is now online.

Comment (dirk-thomas) [reply kinetic/distribution.yaml]: The repository does not contain a release. I guess you haven't invoked `bloom` yet.

File: kinetic/distribution.yaml [hunk 2]
```diff
@@ -1546,6 +1546,25 @@ repositories:
...
+      version: 1.1.0-0
+    source:
+      test_pull_requests: true
```
Comment (dirk-thomas) [inline kinetic/distribution.yaml]: Have you followed the [instructions](http://wiki.ros.org/buildfarm/Pull%20request%20testing) to provide access to this repo or the org unit to the @ros-pull-request-builder user? If not, either remove this line or provide the necessary access as described in the wiki page.

Comment (mbjelonic) [reply kinetic/distribution.yaml]: Thanks for the help! I will do the necessary steps and let you know as soon as I finished.

**Comment by mbjelonic** [comment]
@dirk-thomas : Thank you again for the help!

**Pull request merged**